### PR TITLE
Blog: Release Ember.js Times - Issue #44

### PR DESCRIPTION
**Pull request by jessica-jordan**
To be released on April 27th 2018.

screen shot 2018-04-27 at 01 24 01

### CODE REVIEW

File: source/blog/2018-04-01-the-emberjs-times-issue-44.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: The Ember.js Times - Issue No. 44
+author: all the people
+tags: Recent Posts, Newsletter, Ember.js Times
+alias : "blog/2018/04/01/the-emberjs-times-issue-44.html"
```
Comment (amyrlam) [inline source/blog/2018-04-01-the-emberjs-times-issue-44.md]: should be 04/26

Comment (amyrlam) [reply source/blog/2018-04-01-the-emberjs-times-issue-44.md]: I fixed it here, if you want to merge this as a WIP, whatever workflow is easier for you. https://github.com/emberjs/website/pull/3317

Comment (amyrlam) [reply source/blog/2018-04-01-the-emberjs-times-issue-44.md]: sorry for all the confusion, did that here https://github.com/emberjs/website/pull/3319

**Comment by jessica-jordan** [comment]
Either for this edition or the one one next week, I think we should also add some content for Jen's Stackoverflow blog post - this could also go nicely with some editorial on contributing in general imho

I didn't have time to put something together this week myself, but I could take care of it next week

File: source/blog/2018-04-27-the-emberjs-times-issue-44.md [hunk 2]
```diff
@@ -8,57 +8,74 @@ responsive: true
...
 ---
 
-## [Learning Team corner](https://emberjs.com/blog/2018/04/22/ember-learning-update.html)
+## [Learning Team Corner](https://emberjs.com/blog/2018/04/22/ember-learning-update.html)
 
 The Learning Team met IRL at EmberConf and shared the results of that meeting on [the Ember blog](https://emberjs.com/blog/2018/04/22/ember-learning-update.html). Initiatives for the new year were grouped into two directions - User Support and Marketing.
 
-The new guides app will be launching soon! The main difference is that is in an Ember app, making it easier for Ember folks to contribute. And because of [prember](https://github.com/ef4/prember), it's fully SEO-friendly! Deprecation for the old guides is happening [here](https://github.com/emberjs/guides#notice-this-repository-is-deprecated).
+Some big things are happening over the next few weeks for The Guides! A new app will be launching soon! [Chris
+Manson](https://github.com/mansona) has been working on this project since November, together with a few other
```
Comment (amyrlam) [inline source/blog/2018-04-27-the-emberjs-times-issue-44.md]: Thank yous for adding on here @jenweber! 🙏

**Comment by jessica-jordan** [comment]
I'll 🚀

**Pull request merged**